index.js

```diff
@@ -33,11 +33,13 @@ class Conf {
 
 		options = Object.assign({
 			configName: 'config',
-			fileExtension: 'json'
+			fileExtension: 'json',
+			// Maintain behaviour of 2.0.0 for the new option
```

Comment:
Remove this comment. It doesn't add anything. It's clear from the context what it does.